### DISCUSSION TITLE
[Feedback] Auswirkungen der Mod besser beschreiben

### DISCUSSION THREAD

**Discussion by FrlPetronella** [Taludas/Anno2205-NewFrontiers#6] [Ideas]
Aus der Anleitung geht nicht hervor, was genau die Änderungen an den Spieldaten bewirken. Damit meine ich den Sinn und Zweck des Ordners data (den es bei mir nicht gab) sowie die Auswirkung der Änderung an der engine.ini. Das sollte in der Doku beschrieben sein, damit die Spieler besser einschätzen können, ob sie diese Änderungen in Kauf nehmen wollen oder nicht und welche Auswirkungen das auf bestehende Savegames haben kann.

**Comment by Taludas** [comment]
Erst einmal danke für dein Feedback bezüglich der Dokumentation.
Tatsächlich habe ich (wieder einmal) festgestellt, dass Mod Nutzern jegliche Lese-Kompetenz fehlt, so viele Fragen wie ich bekommen habe, die in der Dokumentation beantwortet sind. Das erstmal vorweg.

Du schlägst nun vor, diese Dokumentation - ich denke der beste Platz dafür wäre in der Installationsanleitung - zu ergänzen um zusätzliche Warnung bzgl. bestehender Spielstände. Da hast du Recht, durch die doofe Savegame Speicherung von 2205 in dem Fall wäre es fatal, eine alte Firma mit der Mod zu starten, da geht alles den Bach runter. Ich denke, dass Werd sich beim nächsten Update dann ergänzen.

Bezüglich der Änderungen im data Ordner. Das interessiert die Nutzer glaube ich einfach nicht. Alle inhaltlichen Änderungen sind auf meiner Website beschrieben. Wie genau das dann funktioniert interessiert glaube ich dir wenigsten. Im Endeffekt überschreiben einfach nur die Dateien aus dem Dta Ordner jene identisch benannten Datein mit gleichem Pfad in den maindata rda- Archiven. ist also kein großes Hexenwerk.
